File: Fundamentos da Programação Orientada a Objetos/ArrayList_OrientadoAObjetos/ex02_v2(ações com base no código do livro)/entidades/Livro.java
package entidades;

public class Livro {
    private String nome;
    private int codigo;
    private String autor;
    private int quantidade;

    public Livro(String nome, int codigo, String autor, int quantidade) {
        this.nome = nome;
        this.codigo = codigo;
        this.autor = autor;
        this.quantidade = quantidade;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public String getAutor() {
        return autor;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public int getQuantidade() {
        return quantidade;
    }

    public void setQuantidade(int quantidade) {
        this.quantidade = quantidade;
    }

    public void exibirLivro() {
        System.out.println("Nome: " + nome);
        System.out.println("Código: " + codigo);
        System.out.println("Autor: " + autor);
        System.out.println("Quantidade: " + quantidade);
    }
}

File: Programação Web Front-End/Lista_Array_POO/exerc_03/js/Livro.js
export default class Livro {
    #nome;
    #codigo;
    #autor;
    #quantidadeEstoque;

    constructor(nome="", codigo="", autor="", quantidadeEstoque=0) {
        this.#nome = nome;
        this.#codigo = codigo;
        this.#autor = autor;
        this.#quantidadeEstoque = quantidadeEstoque;
    }

    get getNome() {
        return this.#nome;
    }

    get getCodigo() {
        return this.#codigo;
    }

    get getAutor() {
        return this.#autor;
    }

    get getQuantidadeEstoque() {
        return this.#quantidadeEstoque;
    }

    exibirLivro() {
        console.log(`Nome: ${this.#nome}` + ` Código: ${this.#codigo}`);
        console.log(`Autor: ${this.#autor}` + ` Quantidade em Estoque: ${this.#quantidadeEstoque}`);
    }
}